### DISCUSSION TITLE
x-text="" is 9 characters !, {{}} is just 4, Use it instead

### DISCUSSION THREAD

**Discussion by RiadhVet** [alpinejs/alpine#4377] [3. Feature ideas]
Hello, 

Would it be possible to add a feature to include values in html using something like double mustaches like in Vue ? that would help and reduce time. Avoid {{}} for projects that already use Vue

**Comment by ekwoka** [comment]
I believe someone made a plugin that would do this. But it could be quite costly, if it isn't also using a directive of some kind...

**Comment by SimoTod** [comment]
Alpine doesn't have a virtual dom, your `{{whatever}}` will show for a fraction of second which is not a great user experience we can justify with fewer keystrokes. 

As mentioned above, there are third party packages to do that if you really need it (I believe it's this one => https://github.com/racha/alpinejs-jsz).
